migrations/helpers/dart_samples_update_helper.py

```diff
@@ -202,7 +168,6 @@ def get_positive_samples(
             # 1. First filter by the start and end dates
             FIELD_CREATED_AT: {"$gte": start_datetime, "$lte": end_datetime},
             FIELD_RESULT: {"$regex": "^positive", "$options": "i"},
-            FIELD_ROOT_SAMPLE_ID: {"$not": {"$regex": "^CBIQA_"}},
```

Comment:
Why remove the control check?

Reply:
To avoid duplication of the control definitions in multiple places: we only want this changing definition to be a part of the fit to pick rules. It does not matter if we add information on a control to DART/for the Beckman process. We know that the fit to pick rules will ensure this is not picked

Reply:
e.g. https://github.com/sanger/crawler/issues/150 adds more control rules, which we'd also have to add to here if we wanted to filter out controls from these samples